recipe/meta.yaml

```diff
@@ -1,26 +1,32 @@
 {% set name = "cryptography-vectors" %}
-{% set version = "43.0.0" %}
+{% set version = "43.0.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/cryptography_vectors-{{ version }}.tar.gz
-  sha256: 5c9d09a732d5433cede1542a96ecd70a80e122af047ee7404bcdf1f3ccb8e702
+  sha256: ff6a885265f484e8907277614b73e9c774b01658623ad763dde2858188e486b4
 
 build:
   skip: true  # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - maturin >=1,<2
   host:
     - python
     - pip
+    - setuptools !=74.0.0,!=74.1.0,!=74.1.1
+    - wheel
```

Comment:
`flit-core` is the build system, why are `setuptools` and wheel` needed?

Reply:
I didn't know `flit-core` was a build system.